contrib/pgo-lto/Make.user.pgo-lto

```diff
@@ -0,0 +1,54 @@
+.PHONY: top clean
```

Comment:
If you add
```suggestion
# -*- mode: makefile -*-
.PHONY: top clean
```
we get the right syntax highlighting on GitHub (and I get the right mode in Emacs :upside_down_face:)

Reply:
Actually this was supposed to be an ordinary Makefile